ledger/accountdb.go

```diff
@@ -1006,7 +998,15 @@ func writeCatchpointStagingBalances(ctx context.Context, tx *sql.Tx, bals []norm
 			if err != nil {
 				return err
 			}
+		} else if err.Error() == "UNIQUE constraint failed: catchpointbalances.address" {
```

Comment:
```
	// address exists: overflowed account record: find addrid
	var sqliteErr sqlite3.Error
	if errors.As(err, &sqliteErr) && sqliteErr.Code == sqlite3.ErrConstraint && sqliteErr.ExtendedCode == sqlite3.ErrConstraintUnique  {
		...
	}
```

Reply:
thanks, I was having trouble finding what the error type was